app/Http/Controllers/PatientController.php

```diff
@@ -87,9 +120,10 @@ public function edit(Patient $patient)
      * @param  \App\Models\Patient  $patient
      * @return \Illuminate\Http\Response
      */
-    public function update(UpdatePatientRequest $request, Patient $patient)
+    public function update(UpdatePatientRequest $request, $id)
     {
-        //
+        $this->patientInterface->update($request, $id);
+        return redirect('/patient');
```

Comment:
Why patient's home screen?
I think that to show redirect patient's list!
`return redirect()->route('patients.list');`

Reply:
patient ka thu ko thu update lote pyi yin home page ko pe pyan thwar ya ml htin loh pr. Admin anay ne so yin tot list ko pyan pya ya mr pr. bl lo lote ya ma ll ako.

Reply:
SHOW page ko pe pyan pya tr kaung ml  htin tl naw ako. patient ka thu ye list ko ma myin ya pr bu

Reply:
Ok, I understood. pls, redirect to patient's list screen at now.